core/src/test/java/org/apache/iceberg/V2TableTestBase.java

```diff
@@ -1,29 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package org.apache.iceberg;
-
-import java.util.Arrays;
-import java.util.List;
-
-public class V2TableTestBase extends TestBase {
```

Comment:
Does removing this test class reduce test coverage?

Reply:
Thanks for checking this. I have two reasons to remove this class; this test base is only inherited from one class like `TestRowDelta`, and this test base has  a `@Parameters` annotated part, and its inherited class (`TestRowDelta`) has also `@Parameters`, so the `@Parameters` annotated part in this test base needs to be removed after updating to JUnit 5.11.1. If this class should be kept, please let me know.